src/Examine/LuceneEngine/Providers/LuceneIndex.cs

```diff
@@ -408,14 +467,44 @@ private void CreateNewIndex(Directory dir)
                     //unlock it!
                     IndexWriter.Unlock(dir);
                 }
+
                 //create the writer (this will overwrite old index files)
                 writer = new IndexWriter(dir, FieldAnalyzer, true, IndexWriter.MaxFieldLength.UNLIMITED);
```

Comment:
This indexwriter deletes the files. Is that the intent?

Reply:
Seems to be

Reply:
@nzdev yes it is intended.